### ISSUE TITLE
Malware detected by my ISP\cpanel provider

### ISSUE THREAD

**Issue by pmorgenrood** [mpdf/mpdf#332]
My website was suspended because one of the example files from latest release was doing something dodge. I then received this email from my provider:
Hello,
The domain was suspended due to the suspicious files found on your account. We have removed the infected file and currently unsuspended the domain . Please scan the files from your local system and re-upload the fresh copy.

PATH:          "/home/aboquic/public_html/" "/home/adamsadamsgradre/public_html/" "/home/adhirr0/public_html/" "/home/adsventu/public_$
RANGE:         1 days
TOTAL FILES:   19601
TOTAL HITS:    1
TOTAL CLEANED: 0

SUSPENDED ACCOUNTS:
petermor

FILE HIT LIST:
{HEX}php.exe.globals.409 : /home/petermor/public_html/mpdf61/examples/example61_new_mPDF_v6-0_features.php => /usr/local/maldetect/qua$
    ===============================================
Linux Malware Detect v1.5 < proj@rfxn.com >

--
Thanks

Regards

Cybersmart Hosting Support
    -------------------
I hope someone can benefit from this info.

**Comment by Klap-in** [comment]
Your version of the example file was still equal to the original?

Does this detection tool provide any guidance to what it dislikes? Because false positives are not uncommon...

The example files are not intended for production. Therefore its code has already moved to https://github.com/mpdf/mpdf-examples

**Comment by pmorgenrood** [comment]
Hi Klap-in, This detection tool is very vague and I have requested more info from my ISP. 

I downloaded using url: [](https://github.com/mpdf/mpdf/releases/download/v6.1.0/01-mPDF-v6.1.0.zip)
extracted the new release which I can see now was zipped with example folder and I didn't check this at the time so just started using mpdf on the site. I have never touched the example files so yes it was as per the original.

Will post comment if\when my provider gives more info on this.
